test/e2e/framework/statefulset_utils.go

```diff
@@ -613,3 +796,31 @@ func NewStatefulSet(name, ns, governingSvcName string, replicas int32, statefulP
 func SetStatefulSetInitializedAnnotation(ss *apps.StatefulSet, value string) {
 	ss.Spec.Template.ObjectMeta.Annotations["pod.alpha.kubernetes.io/initialized"] = value
 }
+
+var statefulPodRegex = regexp.MustCompile("(.*)-([0-9]+)$")
+
+func getStatefulPodOrdinal(pod *v1.Pod) int {
```

Comment:
The package makes use of the term `StatefulPod` extensively. This naming is consistent.